datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -193,6 +196,11 @@ type Dataset implements Entity {
     """
     schema: Schema
 
+    """
```

Comment:
This is a nit, but since we chose to rebrand SchemaMetadata as "Schema" here (everything is metadata so that suffix isn't very enlightening), we should probably be consistent and call this "editableSchema". Maybe or maybe not, just putting out for consideration.

Reply:
I like the consistency here for grep-ability- what is the motivation for dropping the suffix?